html_sanitizer.rst

```diff
@@ -699,7 +698,7 @@ element (even if the original one didn't contain a ``rel`` attribute):
 
         $postSanitizer = new HtmlSanitizer(
             (new HtmlSanitizerConfig())
-                ->forceAttribute('a', 'rel', 'noopener noreferrer')
+                ->forceAttribute('a', ['rel' => 'noopener noreferrer'])
```

Comment:
Here, the parameters are just character strings in the `HtmlSanitizerConfig` class, [see here](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/HtmlSanitizer/HtmlSanitizerConfig.php#L375)

Reply:
oops, good catch, reverted